network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/CalculatedBus.java

```diff
@@ -147,6 +148,40 @@ public Bus setAngle(double angle) {
         return this;
     }
 
+    @Override
+    public double getFictitiousP0() {
+        return voltageLevelResource.getAttributes().getNodeToFictitiousP0().values().stream().reduce(0.0, Double::sum);
+    }
+
+    @Override
+    public Bus setFictitiousP0(double p0) {
+        if (voltageLevelResource.getAttributes().getNodeToFictitiousP0() != null) {
+            voltageLevelResource.getAttributes().getNodeToFictitiousP0().replaceAll((k, v) -> 0.0);
+        }
+        getVoltageLevel().getNodeBreakerView().setFictitiousP0(voltageLevelResource.getAttributes().getNodeToFictitiousP0().keySet().stream()
+                .findFirst()
+                .orElseThrow(() -> new PowsyblException("Bus " + id + " should contain at least one node")),
+                p0);
+        return this;
+    }
+
+    @Override
+    public double getFictitiousQ0() {
+        return this.voltageLevelResource.getAttributes().getNodeToFictitiousQ0().values().stream().reduce(0.0, Double::sum);
+    }
+
+    @Override
+    public Bus setFictitiousQ0(double q0) {
+        if (voltageLevelResource.getAttributes().getNodeToFictitiousQ0() != null) {
+            voltageLevelResource.getAttributes().getNodeToFictitiousQ0().replaceAll((k, v) -> 0.0);
+        }
+        getVoltageLevel().getNodeBreakerView().setFictitiousP0(voltageLevelResource.getAttributes().getNodeToFictitiousQ0().keySet().stream()
```

Comment:
setFictitiousQ0
voltageLevelResource.getAttributes().getNodeToFictitiousQ0() can be null, I think ...